src/main/java/ch/njol/skript/expressions/ExprRepeat.java

```diff
@@ -0,0 +1,80 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Repeat String")
+@Description("repeats a given string, a number of times")
+@Examples({"\"Hello, World! \" repeated 2 times", "nl repeated 100 times"})
+@Since("INSERT VERSION")
+public class ExprRepeat extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprRepeat.class, String.class, ExpressionType.COMBINED, "%string% repeated %integer% time[s]");
```

Comment:
Do you think supporting multiple strings would be useful?

Reply:
I won't deny it crossed my mind and just said screw it don't wanna put that effort into it rn. However, IDK how useful it would be if implemented as 
> `repeat x and y repeated 3 times` would be `xxx and yyy`

I don't see much gain from it as most of the time when you repeat something you tend to use one value and just repeat it either a set number of times or a number depending on another input such as an argument never really needing to modify 2 values at a time with each normally being their own separate lengths

Reply:
Tho I could 100% add it incase people find a use for `nl, nl and nl repeated 10 times` for 33 total new lines

Reply:
Added support for multiple strings